tests/testthat/test-query-describe.R

```diff
@@ -0,0 +1,84 @@
+test_that("describe(it()) with 2 it()s can be matched", {
```

Comment:
At a very high-level, the tests for `describe(it())` and `test_that()` come from a collection of informal tests I've been using to develop queries in the tree-sitter playground.

For `describe(it())`, I'm just drawing on the official examples from testthat.